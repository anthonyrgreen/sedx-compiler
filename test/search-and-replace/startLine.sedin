0first line
second line
2third line
fourth line