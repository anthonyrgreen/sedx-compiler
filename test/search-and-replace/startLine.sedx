let
  myStartLine := '${startLine(), capture(anyOf('0-9')) as firstNum}'
match
${capture(myStartLine()) as myStartLine}
substitute
INSERT! ${myStartLine.firstNum}