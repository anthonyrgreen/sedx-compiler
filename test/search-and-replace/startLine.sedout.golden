INSERT! 0first line
second line
INSERT! 2third line
fourth line